dependencies.yml

```diff
@@ -1,2 +1,4 @@
 - src: carlosbuenosvinos.ansistrano-deploy
   version: 2.7.0
+- src: donat-b.restic
+  version: 0.4
```

Comment:
⚠️ **We still need to use our fork, waiting for a couple of PRs to get merged** ⚠️